version.php

```diff
@@ -25,7 +25,7 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version  = 2016020103;
+$plugin->version  = 2016020104;
```

Comment:
You don't need to bump the version number if there aren't database changes, I bump it on every release

Reply:
I did because there are a new language key and then is easy to clean it. But I can revert it.